internal/utils_test.go

```diff
@@ -1152,3 +1166,66 @@ func TestVerifyManifestPath(t *testing.T) {
 		)
 	}
 }
+
+func TestProcessKustomizeDir(t *testing.T) {
+	baseDirectory, err := filepath.EvalSymlinks(t.TempDir())
+	if err != nil {
+		t.Fatalf("Failed to evaluate symlinks for the base directory: %v", err)
+	}
+
+	// Set up directory structure, with 'workingdir' as target directory:
+	// baseDirectory (t.TempDir())
+	// └── kustomizedir
+	kustomizeDir := path.Join(baseDirectory, "kustomizedir")
+
+	err = os.Mkdir(kustomizeDir, 0o777)
+	if err != nil {
+		t.Fatalf("Failed to create the directory structure %s: %v", kustomizeDir, err)
+	}
+
+	// Create files in baseDirectory/kustomizedir
+	manifestPaths := map[string]string{
+		"kustomization.yaml": `
+resources:
+- configmap.yaml
+- https://github.com/dhaiducek/policy-generator-plugin/examples/input-kustomize/?ref=support-kustomize
```

Comment:
This is where the peer example PR comes in:
- #66

Reply:
Ah, after 66 is merged, this would be updated to point to the stolostron repo (and main branch)?

Reply:
Correct. I thought about setting up another repo for it, but that seemed unnecessary.